tests/tools/test_metrics_3g.py

```diff
@@ -0,0 +1,121 @@
+from typing import TYPE_CHECKING
+
+import numpy as np
+import pandas as pd
+import pertpy as pt
+import pytest
+from anndata import AnnData
+from pertpy.tools._metrics_3g import (
+    compare_class,
+    compare_de,
+    compare_distance,
+    compare_knn,
+)
+
+
+@pytest.fixture
+def test_data():
+    rng = np.random.default_rng()
+    X = rng.normal(size=(100, 10))
+    Y = rng.normal(size=(100, 10))
+    C = rng.normal(size=(100, 10))
+    return X, Y, C
+
+
+@pytest.fixture
+def compare_de_adata():
+    rng = np.random.default_rng()
```

Comment:
You can also use this one :) https://github.com/scverse/pertpy/blob/main/tests/conftest.py#L19